fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
> at least one non-darwin machine the install becomes benign, would be enough?

Yes, pretty much. Out of an abundance of caution I'll verify it myself on some VMs before publishing once you've got it to where you think it's shippable.

> `spawn` on 0.10 does not support the `stdio: "inherit"` option

🤔 it's in the docs: 
https://nodejs.org/dist/latest-v0.10.x/docs/api/child_process.html#child_process_child_process_spawn_command_args_options